renew.c

```diff
@@ -188,9 +192,10 @@ output_creds(krb5_context context, krb5_creds *target_creds)
 	krb5_auth_context auth_context = NULL;
 	krb5_creds **creds = NULL, **c;
 	char *encoded = NULL;
-	krb5_data _creds_data, *creds_data = &_creds_data;
+	krb5_data *creds_data;
```

Comment:
I fail to see how the change of the data type relates to the lifetime setting, and frankly can't see the reason for the change.

Reply:
After that time, I can not see it either. I ran some tests and it really seems to be unnecessary. Reverted.